### PR TITLE
[mdevents] Add template for the mdeventsplugin

### PR DESCRIPTION
**Pull request by elfiesmelfie**
Template included for the mdevents plugin.
All the config values are optional, and so there is no
default/main/mdevents.yml included.
Config values are documented in README.rst

Support for mdevents plugin is not yet included in the container used for
testing with molecule, so tests are not included.
Once the container supports mdevents, then the plugin config can be tested
by modifying the plugin lists in tests/test.yml and
molecule/default/converge.yml.

### CODE REVIEW

File: templates/mdevents.conf.j2 [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+   Interval {{ collectd_plugin_mdevents_interval }}
+</LoadPlugin>
+{% endif %}
+
+<Plugin "mdevents">
+{% if collectd_plugin_mdevents_event is defined %}
+  Event "{{ collectd_plugin_mdevents_event }}"
+{% endif %}
+{% if collectd_plugin_mdevents_ignore_event is defined %}
+  IgnoreEvent {{ collectd_plugin_mdevents_ignore_event | string | lower }}
```
Comment (mrunge) [inline templates/mdevents.conf.j2:14]: shouldn't these event being encapsulated as strings in ticks? "" missing?

File: templates/mdevents.conf.j2 [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{% endif %}
+{% if collectd_plugin_mdevents_array is defined %}
+  Array "{{ collectd_plugin_mdevents_array }}"
+{% endif %}
+{% if collectd_plugin_mdevents_ignore_array is defined %}
+  IgnoreArray {{ collectd_plugin_mdevents_ignore_array | string | lower }}
```
Comment (mrunge) [inline templates/mdevents.conf.j2:20]: same here?

File: templates/mdevents.conf.j2 (hunk 1 above)
Comment (elfiesmelfie) [reply templates/mdevents.conf.j2:14]: IgnoreEvent is a boolean, which doesn't need to be encapsulated as a string

File: templates/mdevents.conf.j2 (hunk 2 above)
Comment (elfiesmelfie) [reply templates/mdevents.conf.j2:20]: IgnoreArray is a boolean, which doesn't need to be encapsulated as a string.

In some cases (IIRC, processes plugin). I see warnings about passing booleans as strings, which state that this is/will be deprecated soon.

**Review by mrunge — APPROVED** [review]
Okay, thank you.

**Pull request merged**